aten/src/ATen/native/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -67,7 +67,9 @@ at::Tensor PackedLinearWeight::apply_dynamic_impl(at::Tensor input) {
       /*qmin=*/is_signed ? -(1 << (precision - 1)) : 0,
       /*qmax=*/
       is_signed ? ((1 << (precision - 1)) - 1) : (1 << precision) - 1,
-      /*preserve_sparsity=*/false);
+      /*preserve_sparsity=*/false,
+      /*force_scale_power_of_two=*/false,
+      /*reduce_range=*/reduce_range);
```

Comment:
The logic in choose quantization params for reduce range is flaky. It works only for unsigned values: The logic currently is:
``` c++  
if (reduce_range) {
    qmin = 0;
    qmax = 127;
  }
we should have:
if (reduce_range)
 {
    qmin = qmin/2; // (0 ->0 and -128 -> -64)
    qmax = qmax/2; // Integer division (255->127 and 127 -> 63)
  }
```